lib/formatio.test.js

```diff
@@ -525,4 +525,29 @@ describe("formatio.ascii", function () {
             assert.equals(str, "[object global]");
         });
     });
+
+    describe("BigInt", function () {
+        if (typeof BigInt === "undefined") {
+            return;
```

Comment:
Better to use Mocha's explicit support for skipping tests based on a condition. Otherwise you will never know that the tests are skipped from the output.

```javascript
before(function () {  
    if (typeof BigInt === "undefined") { 
        this.skip();  
    }    
});
```

Will print 
```
...
    global object
      ✓ formats global object as toString
    BigInt
      - formats 0n
      - formats positive values
      - formats negative values
```

Reply:
Wanted to use this, but saw the usage of `return` in some other tests therefore I did it like this to be consistent. I like your solution and will provide another PR where I will clean other places using `return` instead of `this.skip()`